packages/diracx-web-components/components/DashboardLayout/DashboardDrawer.tsx

```diff
@@ -70,6 +70,8 @@ export default function DashboardDrawer(props: DashboardDrawerProps) {
   const [popAnchorEl, setPopAnchorEl] = React.useState<HTMLElement | null>(
     null,
   );
+  const [renamingItemId, setRenamingItemId] = useState<string | null>(null);
+  const [renamingGroupId, setRenamingGroupId] = useState<string | null>(null);
   const [renameValue, setRenameValue] = React.useState("");
```

Comment:
Do you need to add a type here? (I would have imagined ESLint complaining about it)

Reply:
You mean at this line specifically?

https://github.com/DIRACGrid/diracx-web/blob/00b3ecb468249be295fa31e3c012dfe6e581ac2c/packages/diracx-web-components/components/DashboardLayout/DashboardDrawer.tsx#L73

I think ESLint does not complain because the type is inferred from the initial value, which is a string.